.github/actions/test/action.yml

```diff
@@ -0,0 +1,10 @@
+runs:
```

Comment:
Again, this action should have a `name:` and `description:`

Reply:
[2a8b4c8](https://github.com/DemocracyClub/dc-data-baker/pull/23/commits/2a8b4c88094215255395c6619a21b3d8da52013c)